.github/workflows/release.yml

```diff
@@ -0,0 +1,80 @@
+name: Push Test Release
+
+on:
+  # [ Note: Manually Trigger the Workflow ]
+  # 1. Go to Actions under pytorch/data repo
+  # 2. In the left sidebar, click the workflow you want to run
+  # 3. Above the list of workflow runs, select Run workflow
+  # 4. Use the Branch dropdown to select the release/* branch
+  # 5. Click Run workflow
```

Comment:
Adding notes here. @VitalyFedyunin 
Manually trigger needs to wait until this PR landed.